src/Sulu/Bundle/AdminBundle/Command/UpdateBuildCommand.php

```diff
@@ -268,23 +273,46 @@ private function doManualBuild(SymfonyStyle $ui): int
             return static::EXIT_CODE_ABORTED_MANUAL_BUILD;
         }
 
-        $ui->title('Start manual build ...');
+        $conflictingDirectories = [];
+        foreach ([
+             $this->projectDir . '/node_modules',
+             \dirname(__DIR__, 5) . '/node_modules',
+        ] as $conflictingDirectory) {
+            $renamedConflictingDirectoryName = $conflictingDirectory . '.bak';
+            if ($this->filesystem->exists($conflictingDirectory)) {
+                $this->filesystem->rename($conflictingDirectory, $renamedConflictingDirectoryName);
+            } elseif (!$this->filesystem->exists($renamedConflictingDirectoryName)) {
```

Comment:
think this would be easier to understand if we change the logic to 
```
if ($this->filesystem->exists($conflictingDirectory)) {
    $ui->writeln(\sprintf('Renaming conflicting directory "%s"', $conflictingDirectory));
    $this->filesystem->rename($conflictingDirectory, $renamedConflictingDirectoryName);

...

if ($this->filesystem->exists($renamedConflictingDirectoryName)) {
    $conflictingDirectories[$conflictingDirectory] = $renamedConflictingDirectoryName;
```

Reply:
did update the code